### PR TITLE
updated the page.tsx file

### PR DESCRIPTION
**Pull request by AYassin01**

### CODE REVIEW

**Comment by alexmazaltov** [comment]
@AYassin01 just checked this PR. And I understand we missed this contribution during weekend.

Please check the updates in source repository. Then please update your fork.
If there is a place to put your updates. 

Let me know if i can support you.

File: src/app/page.tsx [hunk 1]
```diff
@@ -169,15 +137,17 @@ export default function Chat() {
...
-                  {/* Generate Joke button */}
                   <button
                     className="bg-blue-500 hover:bg-blue-700 text-white font-bold py-2 px-4 rounded disabled:opacity-50"
-                    disabled={isLoading || !state.genre || !state.tone}
+                    disabled={isLoading || !state.genre || !state.tone || !state.topic}
                     onClick={() =>
                       append({
                         role: "user",
-                        content: `Generate a ${state.genre} story in a ${state.tone} tone`,
+                        content: `Generate a ${state.genre} joke in a ${state.tone} tone about ${state.topic}`,
```
Comment (ionescu77) [inline src/app/page.tsx:150]: I can see this change basically implemented `topic` in the message sent to the backend AI. I have checked `main` branch and this change is already implemented in line 110 `sendRequestToOpenAI`

**Review by ionescu77 — COMMENTED** [review]
Changes already present in the main branch. We should close this PR

File: src/app/page.tsx (hunk 1 above)
Comment (ionescu77) [reply src/app/page.tsx:150]: https://github.com/LuisJoseSanchez/encode-club-ai-and-gpt-bootcamp-q3-homework-week2/blob/935b9388e747eb4afd66ef56831bc4ba26ab31fe/src/app/page.tsx#L110

**Pull request closed without merging**